packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/more-menu.js

```diff
@@ -57,9 +57,9 @@ export default function ScreenNavigationMoreMenu( props ) {
 							>
 								{ __( 'Duplicate' ) }
 							</MenuItem>
-						</MenuGroup>
-						<MenuGroup>
 							<MenuItem
+								isDestructive
+								isTertiary
```

Comment:
Do we need `isTertiary` here? The other menu items I checked don't have this prop, and it adds different background on hover.

Reply:
> Do we need isTertiary here?

I think `isTertialy` prop is necessary. Because without this prop, the border will be drawn even if there is no focus.

### isTertialy=false

https://github.com/WordPress/gutenberg/assets/54422211/ad364a05-1fc7-4549-8ff2-0d01b50d85b6

### isTertiary=true

https://github.com/WordPress/gutenberg/assets/54422211/5cfd6e1b-e515-4e33-8548-8f3c7c65972a

I checked the code base again and noticed that there is only one Delete button that does not have the `isTertiary` prop. It's the page delete button. Therefore, I added the `isTertiary` prop to this button as well.

My feeling is that the default red border is unnecessary, what do you think?

Reply:
> My feeling is that the default red border is unnecessary, what do you think?

I'm not a designer :) but I'd say I agree it's unnecessary. Thanks for working on this ❤️

Reply:
Marking for design feedback. The "tertiary" variation changes the background on hover, which doesn't happen with other buttons. It could be out of place.

Cc @jameskoster, @richtabor

Reply:
Thanks for the ping. 

Neither of these seem quite right to me... The $theme color background when hovering destructive tertiary buttons looks a bit strange with the red text. But I also agree that the border that appears when the button is not tertiary seems heavy-handed and unnecessary.

I'd be inclined to update the latter. IE remove the `box-shadow` applied to `.components-button.is-destructive:not(.is-primary):not(.is-secondary):not(.is-tertiary):not(.is-link)`. Then we can remove `isTertiary` here.

When a destructive button _with_ an outline is required, `isSecondary` can handle that use case. 

Side note: I've seen it suggested elsewhere to remove the red styling on destructive buttons altogether, as it is not a globally appropriate color. Still, getting everything into a consistent state from which we can later refine seems like a good change to me.

Reply:
> remove the `box-shadow` applied to `.components-button.is-destructive:not(.is-primary):not(.is-secondary):not(.is-tertiary):not(.is-link)`. Then we can remove `isTertiary` here.

This appears to be the ideal approach. Indeed, it is strange to have a border by default when in distractive state. However, should this be submitted as a separate PR for review by the component team?

Reply:
Thank you, then I would like to create a separate PR for the button component first.